integrated_channels/integrated_channel/exporters/learner_data.py

```diff
@@ -239,7 +257,7 @@ def _collect_grades_data(self, enterprise_enrollment, course_details):
             completed_date = now
             grade = self.grade_passing
 
-        # Otherwise, the course is still in progress
+        # Otherwise, the course is still in-progress
```

Comment:
@UmanShahzad This looks like a false positive to me :)

Reply:
Ah, yeah :p